packages/moonstone/EditableIntegerPicker/EditableIntegerPicker.js

```diff
@@ -79,8 +87,8 @@ const EditableIntegerPickerBase = kind({
 		decrementIcon: PropTypes.string,
 
 		/**
-		 * When `true`, the EditableIntegerPicker is shown as disabled and does not generate `onChange`
-		 * [events]{@link /docs/developer-guide/glossary/#event}.
+		 * Disables the EditableIntegerPicker and does not generate `onChange`
+		 * [events]{@glossary event}.
```

Comment:
This change undoes the `@link` change above.

Reply:
not sure what happened here :disappear: